test/expect/TestJit.test_cpp.expect

```diff
@@ -1,3 +1,32 @@
+testBlocks
+graph(%a : UNKNOWN_TYPE
+      %b : UNKNOWN_TYPE
+      %c : UNKNOWN_TYPE) {
+  %2 : UNKNOWN_TYPE = add[alpha={1}](%a, %b)
+  %4 : UNKNOWN_TYPE = If(%c)
+    block0() {
+      %5 : UNKNOWN_TYPE = add[alpha={1}](%2, %2)
+      -> (%5)
+    }
+    block1() {
+      %6 : UNKNOWN_TYPE = add[alpha={1}](%b, %2)
+      -> (%6)
+    }
+  return (%4);
+}
+
+graph(%a : UNKNOWN_TYPE
+      %b : UNKNOWN_TYPE
+      %c : UNKNOWN_TYPE) {
+  %2 : UNKNOWN_TYPE = add[alpha={1}](%a, %b)
+  %4 : UNKNOWN_TYPE = If(%c)
```

Comment:
Currently, we have a convention that capitalized is ONNX, and uncapitalized is not-ONNX. Do we want to preserve this convention? (This also applies to your other PRs.)

Reply:
Do we? `FusionGroup`? `GraphExecutor`? `ReplaceIfUndefined`?

Reply:
I admit that we have let this invariant languish, but we have code that assumes it:

```
      // NB: Upper-case is ONNX, lower-case is ATen.  If we want to be more
      // robust, need to explicitly flag operators as ONNX or ATen
      if (!isupper(n[0])) {
        FAIL_EXPORT("Couldn't export operator " + n);
      }
```

Reply:
That's quite strict, because that implies that we have only ONNX and ATen ops in our IR

Reply:
This is just a test, so don't worry about the name.  I will update the test when we actually add support for something like "If" in the interpreter.